### ISSUE TITLE
Doc example not compiling

### ISSUE THREAD

**Issue by joerodrig3** [heroku/react-refetch#73]
I've been hacking at this for a while now, but for some reason can't get the example from the docs below to compile:
```
connect(({ name }) => ({
   foosFetch: '/foos',
   createFoo: name => {
     method: 'POST',
     url: '/foos',
     andThen: () => {
       foosFetch: {
         url: '/foos',
         refreshing: true
       }
     }
   }
 }))
```

The error message given is:
```
Module build failed: SyntaxError: /Users/Joe/Documents/demo/client/app/bundles/demo/startup/Root.jsx: Unexpected token (108:8)
18:30:26 client.1 |   106 |    createFoo: name => {
18:30:26 client.1 |   107 |      method: 'POST',
18:30:26 client.1 | > 108 |      url: '/foos',
18:30:26 client.1 |       |         ^
18:30:26 client.1 |   109 |      andThen: () => {
18:30:26 client.1 |   110 |        foosFetch: {
18:30:26 client.1 |   111 |          url: '/foos',
```

In my own code, I've been trying to make it possible to delete a record and then refresh the data , and it seems as though the example above would allow me to do that, but it doesn't seem to compile. The other snippets given do compile fine

I've tried updating all of my dependencies,restarting my server and a whole bunch of different combinations, but I've only been able to get things to work by wrapping foosFetch and createFoo in parenthesis. This doesn't have the same functionality though and prevents me from modifying the "name" props, which prevents me from refetching automatically.

My work around so far has been to fetch data using refetch but then in my root react component I have to handle all of the logic for updating there:
ie.
```
  _destroy(id) {
    this.props.destroy(id);
    this.props.refresh();
  }
```

**Comment by eyalw** [comment]
you're missing parantesis:

```javascript
connect(({ name }) => ({
   foosFetch: '/foos',
   createFoo: name => {
     method: 'POST',
     url: '/foos',
     andThen: () => {
       foosFetch: ({ // <-- here
         url: '/foos',
         refreshing: true
       }) // <-- here
     }
   }
 }))
```

**Comment by joerodrig3** [comment]
Ah I see, thank you. I was able to get this working last night. Just for reference for everyone, what @eyalw said above is the correct syntax if you're chaining a refresh call.

If you want to chain anything else, you wouldn't the parenthesis, ie. modifying props(see issue https://github.com/heroku/react-refetch/issues/47).

I'll submit a quick PR to add the parenthesis to the docs.

**Comment by ryanbrainard** [comment]
Fixed by #47. Sorry about the trouble here!
